interfaces/functions.yaml

```diff
@@ -171,6 +174,13 @@ igraph_realize_degree_sequence:
         OUT GRAPH graph, VECTOR_INT out_deg, OPTIONAL VECTOR_INT in_deg=NULL,
         EDGE_TYPE_SW allowed_edge_types=SIMPLE, REALIZE_DEGSEQ_METHOD method=SMALLEST
 
+igraph_circulant:
+    PARAMS: OUT GRAPH graph, INTEGER n, VECTOR_INT shifts, BOOLEAN directed
+
+igraph_weighted_sparsemat:
+    PARAMS:
+        OUT GRAPH graph, SPARSEMAT A, BOOLEAN directed, CSTRING attr, BOOLEAN loops=False
```

Comment:
Note to ourselves: this reflects the current state correctly, but we will probably have to update `igraph_weighted_sparsemat` soon to be consistent with `igraph_weighted_adjacency` in the sense that it does not take an attribute _name_ but a vector (in which the weights will be stored). See `CHANGELOG.md` for the reasons.